Makefile

```diff
@@ -12,6 +12,7 @@ clean: clean_openwrt clean_feeds clean_binaries clean_keys
 # Building OpenWRT
 $(DIR__CI)/patched:
 	git submodule init openwrt;git submodule update --remote; \
+	cp $(DIR__CI)/feeds.conf.default $(DIR__OPENWRT)/feeds.conf.default; \
```

Comment:
how about having this file named as feeds.conf instead so that original openwrt feeds.conf.default will remain intact, but feeds.conf does take the precedence.